services/spar/test-integration/Test/Spar/Scim/UserSpec.hs

```diff
@@ -256,7 +256,6 @@ testCreateUserNoIdP = do
     liftIO $ accountStatus brigUserAccount `shouldBe` PendingInvitation
     liftIO $ userEmail brigUser `shouldBe` Just email
     liftIO $ userManagedBy brigUser `shouldBe` ManagedByScim
-    liftIO $ userSSOId brigUser `shouldBe` Just (UserScimExternalId (fromEmail email))
```

Comment:
Don't remove, but check that it's `Nothing`.  If that doesn't work either, leave a FUTUREWORK.

Reply:
Done